src/CMakeLists.txt

```diff
@@ -115,6 +115,9 @@ elseif(MACOSX)
 endif()
 
 if(UMF_BUILD_SHARED_LIBRARY)
+    # Set the runtime search path to the directory containing hwloc library
+    set(CMAKE_INSTALL_RPATH "\$ORIGIN")
```

Comment:
just want to make sure - isn't there a better solution to that problem...? 😉 now, some random value of `$ORIGIN` could make unexpected issues, or something...?

Reply:
we can do similarly like we have for hwloc name change and only set it when required